vm/src/stdlib/time_module.rs

```diff
@@ -58,6 +58,18 @@ pub fn get_time() -> f64 {
     Date::now() / 1000.0
 }
 
+#[cfg(any(not(target_arch = "wasm32"), target_os = "wasi"))]
+pub fn get_time_ns() -> u64 {
+    match SystemTime::now().duration_since(UNIX_EPOCH) {
+        Ok(v) => v.as_nanos() as u64,
+        Err(err) => panic!("Time error: {:?}", err),
```

Comment:
```suggestion
        Err(err) => unsafe { std::hint::unreachable_unchecked() }, // guaranted to be not to be happen with now() + UNIX_EPOCH
```

this hint will remove the branch itself for release build